Rename the module

saipavan10-git opened the pull request:
This PR has changes related to renaming the module.

pbchase reviewed (changes requested):
Could you also replace the action tag? Replace "@TUBE-LABEL-GENERATOR" with "@ZEBRA-LABEL-PRINTER". 

Grep shows it appearing in 6 places in the PR branch. One occurrence is in `examples/tube_label_generator_example_1.xml`. Could also rename that file to `examples/example_1.xml`

saipavan10-git commented:
I updated the module's action tag name and modified the logic that adds the action tag to the table list on the online designer page. I needed to change this because the existing logic doesn't consider if there are no action tag names present with the starting letter of the new tag being added to the lexicographically sorted table. We'll add the new tag name to the beginning of the list. 


![image](https://github.com/user-attachments/assets/05387c56-a09b-47d1-9034-7fc1827d076a)

pbchase reviewed (approved):
Looks good!
